out_writeapi.go

```diff
@@ -49,16 +50,18 @@ type outputConfig struct {
 }
 
 var (
-	ms_ctx    = context.Background()
-	configMap = make(map[int]*outputConfig)
-	configID  = 0
+	ms_ctx              = context.Background()
+	configMap           = make(map[int]*outputConfig)
+	configID            = 0
+	queueRequestScaling = 0
```

Comment:
Suggest renaming this to something like "pendingRequestCountScaleupThreshold".

Also, shouldn't this be per-output instance since the queue size limit can also be defined per instance?

Reply:
Done! Also yes that is correct; I completely missed that!